src/Symfony/Component/Validator/Constraints/AbstractStringContains.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Constraints;
+
+use Symfony\Component\Validator\Constraint;
+
+/**
+ * @Annotation
+ * @Target({"PROPERTY", "METHOD", "ANNOTATION"})
+ */
+abstract class AbstractStringContains extends Constraint
+{
+    /**
+     * @var string|array|null
+     */
+    public $text;
```

Comment:
should we really support array (or iterable ;)) values? compared to using the constraint many times ...